logging.cfg

```diff
@@ -1,5 +1,5 @@
 [loggers]
-keys=root,asaps_logger
+keys=root,models
```

Comment:
This should actually be the fully qualified name `asaps.models` (and the same down further). Making this change may or may not work as expected because in looking at this I realized there is a problem with your import statements. I would say go ahead and change this, whether it works right now or not, and I'll create an issue for the changes to the import statements. Instead of using `asaps.models` you could also just use `asaps`--in fact, I'd recommend it--because then you don't have to add another logger section for each module you create. If, in the future, you decide you need a different logger for some module you can always add it later.

Reply:
Fixed and yeah, it defaulted back to the root logger